### PR TITLE
🐠 Added UpdateUserEmail patch

### PR DESCRIPTION
**Pull request by dylanbrookes**
user.go -> added UpdateUserEmail function

user_test.go -> added TestUpdateUserEmail

<sup> Created from VS Code using [CodeStream](https://codestream.com/?utm_source=cs&utm_medium=pr&utm_campaign=github*com)</sup>

### CODE REVIEW

File: user.go [hunk 1]
```diff
@@ -89,3 +89,28 @@ func (a *App) DeactivateUser(userID string) (*User, error) {
...
+func (a *App) UpdateUserEmail(userID string, email string) (*User, error) {
+	type respUser struct {
+		User User `json:"user"`
+	}
+	var userBody respUser
+
+	response, err := resty.New().R().
+		SetAuthToken(a.Config.APIKey).
+		SetResult(&userBody).
+		SetBody(fmt.Sprintf(`{"email": %v}`, email)).
```
Comment (apobletts) [inline user.go]: I believe the correct way to do this is using a quick struct

type updateBody struct {
      Email string `json:"email"`
}
...
then use that struct in the SetBody

**Review by apobletts — CHANGES_REQUESTED** [review]

**Comment by apobletts** [comment]
@dylanbrookes sorry for the delay on this, but I think we agreed to change all these to updateUser to be able to update any fields in the future.

File: user.go [hunk 2]
```diff
@@ -13,11 +13,14 @@ type User struct {
 	ID            string    `json:"id"`
 	Active        bool      `json:"active"`
 	Email         string    `json:"email"`
+	Phone         string    `json:"phone"`
```
Comment (calhorn95) [inline user.go:16]: *****phone_number

Comment (dylanbrookes) [reply user.go:16]: Resolving this conversation; we've moved to `phone`

**Review by calhorn95 — APPROVED** [review]

**Pull request merged**